.github/workflows/eerelease.yml

```
@@ -0,0 +1,124 @@
+name: Release Ansible Execution Environment
+on:
+  workflow_dispatch:
+    inputs:
+      ee-name:
+        description: >-
+          EE name if it is community-ee-base or community-ee-minimal. Example: community-ee-base
+        required: true
+      ansible-core-version:
+        description: >-
+          Release Version. Example: 11.1.0
+        required: true
+      ee-version:
+        description: >-
+          EE Release Version. Example: 1
+        required: true
+      tag:
+        description: >-
+          Whether to tag the image as latest.
+        type: boolean
+        required: false
+        default: false
+
+env:
+  EE_VERSION: ${{ inputs.ansible-core-version }}-${{ inputs.ee-version }}
+
+jobs:
+  build:
+    name: Build Execution Environment ${{ inputs.ansible-core-version }}-${{ inputs.ee-version }}
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+      packages: write
+
+    steps:
+      - name: Check out images
+        uses: actions/checkout@v4
+        with:
+          repository: ansible-community/images
+          ref: main
+          path: image
+
+      - name: Check out eercheck repo for testing
+        uses: actions/checkout@v4
+        with:
+          repository: anweshadas/eercheck
+          ref: main
+          path: eercheck
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install ansible-builder
+          python3 -m pip install setuptools
+          sudo apt install podman --yes
+          pwd
+          ls -l
+
+      - name: Pull the latest fedora image
+        run: |
+          podman pull fedora:latest
+
+      - name: Build Community EE image
+        working-directory: image/execution-environments/${{ inputs.ee-name }}
+        run: |
+          ansible-builder build --tag ghcr.io/ansible-community/${{ inputs.ee-name }}:${{ env.EE_VERSION }}
+
+      - name: Find the id of the latest EE image
+        run: |
+          echo "IMAGE_ID=$(podman images -q ${{ inputs.ee-name }}:${{ env.EE_VERSION }})">>$GITHUB_ENV
```

Shouldn't it be
```suggestion
          echo "IMAGE_ID=$(podman images -q "ghcr.io/ansible-community/${{ inputs.ee-name }}:${EE_VERSION}")" >> $GITHUB_ENV
```
since the image's name includes `ghcr.io/ansible-community/`? The reminder of the suggestion improves shell handling / security.